lib/views/help/credits.html.erb

```diff
@@ -60,28 +60,31 @@
         maintain the database of public authorities and so much more. Thanks to
         <a href="https://www.whatdotheyknow.com/user/m_bimmler">Michael
         Bimmler</a>,
-        <a href="https://www.whatdotheyknow.com/user/kieran_casey_2">
-        Kieran Casey-McEvoy</a>,
-        <a
-        href="https://www.whatdotheyknow.com/user/gavin_chait/">Gavin Chait</a>,
+        <a href="https://www.whatdotheyknow.com/user/kieran_casey_2">Kieran
+          Casey-McEvoy</a>,
```

Comment:
As you're after consistency, how about:

```suggestion
        Casey-McEvoy</a>,
```

Reply:
Sorry, yes, that should be as per suggestion. Fixed